include/visit_struct/visit_struct.hpp

```diff
@@ -141,8 +252,7 @@ struct visitable<STRUCT_NAME, void> {
     using self_type = STRUCT_NAME;                                                                 \
     VISIT_STRUCT_PP_MAP(VISIT_STRUCT_MEMBER_HELPER_TYPE, __VA_ARGS__)                              \
   }                                                                                                \
-                                                                                                   \
-  static constexpr bool value = true;                                                              \
+  static VISIT_STRUCT_CONSTEXPR const bool value = true;                                           \
```

Comment:
Are you changing this because MSVC has problems with it, or just because we used VISIT_STRUCT_CONSTEXPR elsewhere?

The reason I added VISIT_STRUCT_CONSTEXPR is because in C++14 standard, void functions can be constexpr so all the `apply` functions can be `constexpr`. In C++11 they can't be, `constexpr` functions are just supposed to return a value and not execute statements or really have side effects. So the apply functions should not be `constexpr` there, but `static constexpr bool value = true` is still kosher in C++11, and it's useful so you can make `static_assert` and such using the value.

If it's a problem for some versions of MSVC or something then that's worth considering, but iirc if this is a problem it would be a problem for the intrusive header also and that one did test okay in MSVC 2015.

At least, that's the way I view the purpose. It might be good to rename VISIT_STRUCT_CONSTEXPR or put some notes about why its there, I really intended it to be a "have C++14 CONSTEXPR" option basically.